docs/guide-ar/README.md

```diff
@@ -0,0 +1,205 @@
+ Yii 2.0 الدليل التقني الخاص ببيئة العمل
+===============================
+
+تم تحرير هذا الملف اعتمادا على [الشروط الخاصة بتوثيف ال Yii](http://www.yiiframework.com/doc/terms/).
+
+جميع الحقوق محفوظة
+
+2014 (c) Yii Software LLC.
+
+
+المقدمة
+------------
+
+* [عن بيئة العمل Yii](intro-yii.md)
+* [التحديث من الإصدار 1.1](../guide/intro-upgrade-from-v1.md)
+
+
+البداية من هنا
+---------------
+
+* [ماذا يجب أن تعرف عن بيئة العمل](start-prerequisites.md)
+* [تنصيب ال Yii](../guide/start-installation.md)
+* [تشغيل وتطبيق بيئة العمل](../guide/start-workflow.md)
+* [قل مرحبا - المشروع الأول](../guide/start-hello.md)
+* [العمل مع ال forms](../guide/start-forms.md)
+* [العمل مع قواعد البيانات](../guide/start-databases.md)
+* [إنشاء الشيفرة البرمجية من خلال ال gii](../guide/start-gii.md)
+* [ماذا الآن - الخطوة القادمة](../guide/start-looking-ahead.md)
+
+
+الهيكلية الخاصة بالتطبيق (Application Structure)
+---------------------
+
+* [نظرة عامة عن الهيكلية الخاصة بالتطبيق](../guide/structure-overview.md)
+* [Entry Scripts](../guide/structure-entry-scripts.md)
+* [التطبيقات](../guide/structure-applications.md)
+* [مكونات التطبيقات](../guide/structure-application-components.md)
+* [Controllers](../guide/structure-controllers.md)
+* [Models](../guide/structure-models.md)
+* [Views](../guide/structure-views.md)
+* [Modules](../guide/structure-modules.md)
+* [Filters](../guide/structure-filters.md)
+* [Widgets](../guide/structure-widgets.md)
+* [Assets](../guide/structure-assets.md)
+* [Extensions](../guide/structure-extensions.md)
+
+
+التعامل مع ال requests
+-----------------
+
+* [نظرة عامة عن التعامل مع ال requests](../guide/runtime-overview.md)
+* [Bootstrapping](../guide/runtime-bootstrapping.md)
+* [Routing and URL Creation](../guide/runtime-routing.md)
+* [Requests](../guide/runtime-requests.md)
+* [Responses](../guide/runtime-responses.md)
+* [Sessions and Cookies](../guide/runtime-sessions-cookies.md)
+* [Handling Errors - التحكم بالأخطاء](../guide/runtime-handling-errors.md)
+* [Logging - تسجيل الحركات](../guide/runtime-logging.md)
+
+
+المفاهيم الرئيسية (Key Concepts)
+------------
+
+* [Components](../guide/concept-components.md)
+* [Properties](../guide/concept-properties.md)
+* [Events](../guide/concept-events.md)
+* [Behaviors](../guide/concept-behaviors.md)
+* [Configurations](../guide/concept-configurations.md)
+* [Aliases](../guide/concept-aliases.md)
+* [Class Autoloading](../guide/concept-autoloading.md)
+* [Service Locator](../guide/concept-service-locator.md)
+* [Dependency Injection Container](../guide/concept-di-container.md)
+
+
+التعامل مع قواعد البيانات
+----------------------
+
+* [Database Access Objects](../guide/db-dao.md): Connecting to a database, basic queries, transactions, and schema manipulation
```

Comment:
Not translated yet.

Reply:
Hi @samdark, I keep some of keyword in English since its will be "Literal translation", for example: "Bootstrapping" in Literal translation will be "إلباس الحذاء", and sure its wrong, and if you translate it again, its will be "boot"....and so on...In our culture in programming, much of key word we keep it as it, and translate or describe how its work...etc

Reply:
Note: Other pages, will be added one by one in near future, Im start by translate 2 pages with README as you see in comments.

Reply:
No, that's fine.